test/parallel/test-assert-calltracker-calls.js

```diff
@@ -78,3 +78,39 @@ assert.throws(
   callsNoop();
   tracker.verify();
 }
+
+{
+  function func() {}
+  const tracker = new assert.CallTracker();
+  const callsfunc = tracker.calls(func);
+  assert.strictEqual(callsfunc.length, 0);
+}
+
+{
+  function func(a, b, c = 2) {}
+  const tracker = new assert.CallTracker();
+  const callsfunc = tracker.calls(func);
+  assert.strictEqual(callsfunc.length, 2);
+}
+
+{
+  function func(a, b, c = 2) {}
+  delete func.length;
+  const tracker = new assert.CallTracker();
+  const callsfunc = tracker.calls(func);
+  assert.strictEqual(Object.hasOwn(callsfunc, 'length'), false);
+}
+
+{
+  Object.prototype.get = common.mustNotCall('%Object.prototype%.get');
+  const customPropertyValue = Symbol();
+  function func(a, b, c = 2) {}
+  func.customProperty = customPropertyValue;
+  Object.defineProperty(func, 'length', { get: common.mustNotCall() });
+  const tracker = new assert.CallTracker();
+  const callsfunc = tracker.calls(func);
+  assert.strictEqual(Object.hasOwn(callsfunc, 'length'), true);
+  assert.strictEqual(callsfunc.customProperty, customPropertyValue);
+
+  delete Object.prototype.get;
```

Comment:
```suggestion
  delete Object.prototype.get;
  ArrayIteratorPrototype.next = next;
```

Reply:
The latest changes have been pushed, thank you very much for your review these days, it has benefited me a lot.